src/state/routing/types.ts

```diff
@@ -132,13 +147,34 @@ type URAClassicQuoteResponse = {
 }
 export type URAQuoteResponse = URAClassicQuoteResponse | URADutchOrderQuoteResponse
 
+export type QuickRouteResponse = {
+  tokenIn: {
+    address: string
+    decimals: number
+    symbol: string
+    name: string
+  }
+  tokenOut: {
+    address: string
+    decimals: number
+    symbol: string
+    name: string
+  }
+  tradeType: 'EXACT_IN' | 'EXACT_OUT'
+  quote: {
+    amount: string
+    path: string
+  }
+}
+
 export function isClassicQuoteResponse(data: URAQuoteResponse): data is URAClassicQuoteResponse {
   return data.routing === URAQuoteType.CLASSIC
 }
 
 export enum TradeFillType {
   Classic = 'classic', // Uniswap V1, V2, and V3 trades with on-chain routes
   UniswapX = 'uniswap_x', // off-chain trades, no routes
+  None = 'none', // for preview trades, cant be used for submission
```

Comment:
Although I thought we were going to go with Preview Trades for the naming convention?
```suggestion
  None = 'none', // for quick trades, cant be used for submission
```